extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
 	 * @return Array
 	 */
 	public function getTileSets() {
-		$params = [];
+		$params = [
```

Comment:
AFAIK Zach wanted to always display 50 latest tile set also when using search. I set it up to 49 cause the first item i the list is upload button and it makes 50 items together with it